demo/Gemfile

```diff
@@ -33,9 +33,7 @@ gem "haml-rails"
 group :development, :test do
   gem "citizens-advice-style",
       github: "citizensadvice/citizens-advice-style-ruby",
-      tag: "v9.0.0"
-
-  gem "rubocop-rails", require: false
```

Comment:
Is this because rubocop rails is now required?

Reply:
Yeah citizens-advice-style now bundles rubocop-rails and rubocop-rspec